rero_ils/modules/stats/permissions.py

```diff
@@ -77,3 +83,25 @@ def delete(cls, user, record):
         :return: True if action can be done.
         """
         return False
+
+
+def stats_ui_permission_factory(record, *args, **kwargs):
+    """Permission for stats detailed view."""
+    return record_permission_factory(
+            action='read', record=record, cls=StatPermission)
+
+
+def check_logged_as_admin(fn):
+    """Decorator to check if the current logged user is logged as an admin.
+
+    If no user is connected: return 401 (unauthorized)
+    If current logged user isn't `librarian`: return 403 (forbidden)
+    """
```

Comment:
I'm not certain about the admin part, but that's how I understand it.
```suggestion
    """Decorator to check if the current logged user is an admin.

    If no user is connected: return 401 (unauthorized)
    If current logged user isn't `admin`: return 403 (forbidden)
    """
```